### DISCUSSION TITLE
Spotify credentials get blocked after connecting Spotube to Spotify account

### DISCUSSION THREAD

**Discussion by mbutii** [KRTirtho/spotube#725] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The day after logging into my Spotify account from Spotube, my credentials got blocked in Spotify and could not login anymore neither from the Spotify app or from Spotify web.
I am running Spotube 3.1.1 on Linux.

I had to reset Spotify credentials in order to have my account work again.

### Expected Behavior

I expected that Spotube doesn't break the Spotify account

### Steps to reproduce

1. I installed Spotube
2. I logged in into the Spotify account using the official instructions (e.g. taking the credentials IDs from the cookies in the browser)
3. I played a little bit with Spotube both online and offline
4. The day after I tried using Spotify from the mobile app and discovered that  the app was logged out, and that I could not login anymore with the same credentials

### Operating System

Linux

### Spotube version

3.1.1

### Installation source

Flathub (Flatpak)

**Comment by KRTirtho** [comment]
It can happen if somehow your credentials got leaked. Especially be careful when you copy the cookies. Make sure to clean your clipboard because anything & everything has access to this info in your computer. Even shady websites can get those clipboard data

Also we don't know the nature of Spotify Auth service. Personally I've been using the same account for developing the app which makes over a million requests in a week may be but I never faced any blocking

**Comment by mbutii** [comment]
Thanks. it was indeed as you said. Using stronger credentials solved the issue and I don't get kicked out by the Spotify account anymore

**Comment by jasperweiss** [comment]
My Spotify account gets blocked every time I use Spotube. I use the password reset email to set a new password (generated and stored by an offline password manager) and the next day I'm logged out. I try to log back in using the browser and find that, once again, my credentials no longer work.

**Reply by jasperweiss** [reply]
In fact, password reset emails do not even arrive anymore. It seems my account is deleted completely.

**Reply by kbrandt** [reply]
Same on my side. 
I have created a Spotify trial account for this app only. This account has only been used once. 
But after a while the trial account becomes invalid. It was also not possible to reset the account. Only a Spotify support person was able to help.
